opentreemap/treemap/js/src/lib/geocoderUi.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
```

Comment:
I reread the PR description and I see that this removing the dark background is intentional